clr-k8s-examples/setup_system.sh

```diff
@@ -3,85 +3,159 @@
+}
+
+# permanently disable swap
+function disable_swap() {
+  swapcount=$(sudo grep '^/dev/\([0-9a-z]*\).*' /proc/swaps | wc -l)
 
-sudo mkdir -p /etc/sysctl.d/
-cat <<EOT | sudo bash -c "cat > /etc/sysctl.d/60-k8s.conf"
+  if [ "$swapcount" != "0" ]; then
+    sudo systemctl mask $(sed -n -e 's#^/dev/\([0-9a-z]*\).*#dev-\1.swap#p' /proc/swaps) 2>/dev/null
+  else
+    echo "Swap not enabled"
+  fi
+}
+
+# enable ip forwarding
+function enable_ip_forwarding() {
+  sudo mkdir -p /etc/sysctl.d/
+  cat <<EOT | sudo bash -c "cat > /etc/sysctl.d/60-k8s.conf"
 net.ipv4.ip_forward=1
 EOT
-sudo systemctl restart systemd-sysctl
+  sudo systemctl restart systemd-sysctl
+
+}
 
-#Ensure the modules we need are preloaded
-sudo mkdir -p /etc/modules-load.d/
-cat <<EOT | sudo bash -c "cat > /etc/modules-load.d/k8s.conf"
+# ensure the modules we need are preloaded
+function setup_modules_load() {
+  sudo mkdir -p /etc/modules-load.d/
+  cat <<EOT | sudo bash -c "cat > /etc/modules-load.d/k8s.conf"
 br_netfilter
 vhost_vsock
 overlay
 EOT
+}
 
-# Make sure /etc/hosts file exists
-if [ ! -f /etc/hosts ]; then
-  sudo touch /etc/hosts
-fi
-hostcount=$(grep '127.0.0.1 localhost' /etc/hosts | wc -l)
-if [ "$hostcount" == "0" ]; then
-	echo "127.0.0.1 localhost $(hostname)" | sudo bash -c "cat >> /etc/hosts"
-else
-	echo "/etc/hosts already configured"
-fi
+# ensure hosts file setup
+function setup_hosts() {
+  # Make sure /etc/hosts file exists
+  if [ ! -f /etc/hosts ]; then
+    sudo touch /etc/hosts
+  fi
+  # add localhost to /etc/hosts file
+  hostcount=$(grep '127.0.0.1 localhost' /etc/hosts | wc -l)
+  if [ "$hostcount" == "0" ]; then
+    echo "127.0.0.1 localhost $(hostname)" | sudo bash -c "cat >> /etc/hosts"
+  else
+    echo "/etc/hosts already configured"
+  fi
+}
 
-sudo systemctl daemon-reload
-# This will fail at this point, but puts it into a retry loop that
-# will therefore startup later once we have configured with kubeadm.
-echo "The following kubelet command may complain... it is not an error"
-sudo systemctl enable --now kubelet $RUNNER || true
+# daemon reload
+function daemon_reload() {
+  sudo systemctl daemon-reload
+}
+
+# enable kubelet for $RUNNER
+function enable_kubelet_runner() {
+  # This will fail at this point, but puts it into a retry loop that
+  # will therefore startup later once we have configured with kubeadm.
+  echo "The following kubelet command may complain... it is not an error"
+  sudo systemctl enable --now kubelet $RUNNER || true
+}
 
-#Ensure that the system is ready without requiring a reboot
-sudo swapoff -a
-sudo systemctl restart systemd-modules-load.service
+# ensure that the system is ready without requiring a reboot
+function ensure_system_ready() {
+  sudo swapoff -a
+  sudo systemctl restart systemd-modules-load.service
+}
 
+# add proxy if found
+function setup_proxy(){
 set +o nounset
 if [[ ${http_proxy} ]] || [[ ${HTTP_PROXY} ]]; then
-	echo "Setting up proxy stuff...."
```

Comment:
https://github.com/mvdan/sh

`shfmt -l -w`

Reply:
I will format with shfmt before removing WIP. Thanks!

Reply:
@jascott1 this is the formatting i was referring to?

Reply:
@krsna1729 Sorry if I am missing your point but it looks fixed: https://github.com/clearlinux/cloud-native-setup/pull/141/files#diff-7d66763fb870e9a8078452413d51d662R94

Reply:
@jascott1 I just tried pulling your patch and checked the tool against it. i think half our code is 2-spaces indent and other half is a tab indent. If that is alright with you, can you just run the command that @krsna1729 is suggesting and get that change into this also. I know it is just a cosmetic change, but it is good to make it consistent.

```
ganeshma@ganeshma-lab1 <> cloud-native-setup % hub pr checkout 141
warning: redirecting to https://github.com/clearlinux/cloud-native-setup/
From http://github.com/clearlinux/cloud-native-setup
 * [new ref]         refs/pull/141/head -> setupos
Switched to branch 'setupos'
ganeshma@ganeshma-lab1 <> cloud-native-setup % shfmt -w -l .
clr-k8s-examples/admit-kata/create-certs.sh
clr-k8s-examples/containerd_devmapper_setup.sh
clr-k8s-examples/setup_system.sh
clr-k8s-examples/tests/e2e/run_e2e.sh
metrics/lib/common.bash
metrics/lib/json.bash
metrics/lib/k8s-api.bash
metrics/report/grabdata.sh
metrics/report/makereport.sh
metrics/scaling/k8s_parallel.sh
metrics/scaling/k8s_scale.sh
ganeshma@ganeshma-lab1 <> cloud-native-setup % git status
On branch setupos
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   clr-k8s-examples/admit-kata/create-certs.sh
        modified:   clr-k8s-examples/containerd_devmapper_setup.sh
        modified:   clr-k8s-examples/setup_system.sh
        modified:   clr-k8s-examples/tests/e2e/run_e2e.sh
        modified:   metrics/lib/common.bash
        modified:   metrics/lib/json.bash
        modified:   metrics/lib/k8s-api.bash
        modified:   metrics/report/grabdata.sh
        modified:   metrics/report/makereport.sh
        modified:   metrics/scaling/k8s_parallel.sh
        modified:   metrics/scaling/k8s_scale.sh

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
I ran `shfmt -l -w` and pushed that so hopefully that took care of it.